devops/rpmbuild/extrafiles/basecoin/usr/share/basecoin/key2.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This is something that we should discuss. The key2.json is generated if you run basecoin init but it's a static file. I don't think the binary should generate static files for us, I think it should be supplied separately - in this case the RPM (or deb). The other issue is that the key file is generated into the configuration directory but a key file is more of a personal asset then a configuration for the server. Right now this is in limbo:
1. either I remove the key file from the repo and keep things as is, or
2. I remove the file but we put it into the basecoin repo so the rpm can pull it
3. keep the file.

I don't like the third option, all necessary files for packaging should be part of the source repository. I don't like the first option either because we clutter the configuration folder with unnecessary files.

But the RPM should definitely have this file, preferably installing it out of the way, like in /usr/share/basecoin (as any other application in linux that has extra examples).